diesel/src/lib.rs

```diff
@@ -129,10 +129,10 @@
 #[macro_use]
 extern crate bitflags;
 extern crate byteorder;
+
 #[macro_use]
 extern crate diesel_derives;
-#[doc(hidden)]
-pub use diesel_derives::*;
+pub use diesel_derives::sql_function_proc as sql_function;
```

Comment:
This is missing documentation.

Also i think `diesel::expression` could be a better place for this.